### ISSUE TITLE
In the realtime effects panel, how to reorder effects using the keyboard is not discoverable

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#3504]
### Steps to reproduce

1. Open the realtime effects panel
2.  Add two of more effects
3.  Try to reorder the effects using the keyboard and without consulting any documentation

### Expected behavior

Success.

### Actual behavior

Failure.

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

### Additional context

In other cases in Audacity where items can be reordered (tracks, commands in macros), reordering them using the keyboard is discoverable without needing to read any documentation.

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
